package.json

```diff
@@ -9,6 +9,8 @@
   "author": "DuckDuckGo",
   "license": "MIT",
   "devDependencies": {
-    "grunt": "^0.4.5"
+    "grunt": "^1.0.1",
```

Comment:
You know, I don't remember actually doing this, but I've got this vague recollection of getting an update prompt from Grunt.

Looking at the modules I installed, they only need Grunt 0.4+. I'll revert this so things are consistent.